src/main/java/com/gzs/main/RestMethods.java

```diff
@@ -28,14 +34,17 @@ public String getTerm(@QueryParam("term") String termName) throws JsonProcessing
         if (term1ID != 0) {
             int term2ID = DBMethods.getTranslatedTermID(term1ID);
             if (term2ID != 0) {
-                Term term = DBMethods.getTermByID(term2ID);
-                ObjectMapper mapper = new ObjectMapper();
+                term = DBMethods.getTermByID(term2ID);
                 return mapper.writeValueAsString(term);
             } else {
-                return "<h2>Hello, there is no translation for " + termName + " in dictionary.</h2>";
+                term = new Term();
+                term.setTerm("There is no translation for " + termName + " in dictionary.");

```

Comment:
@DespotOfMorea I'm not sure how good idea is to use `Term` class for transfer of error messages. It might cause confusion on front-end (or with other consumers of this API). It could be better to create separate class for errors and other messages and return instance of it.

Reply:
Created new class (CustomInfo.class) for returning different type of messages and used it in this method.
```diff
            } else {
-                term = new Term();
-                term.setTerm("There is no translation for " + termName + " in dictionary.");
-                return mapper.writeValueAsString(term);
+                CustomInfo infoMsg = new CustomInfo('i',"There is no translation for " + termName + " in dictionary.");
+                return mapper.writeValueAsString(infoMsg);
            }
        } else {
-            term = new Term();
-            term.setTerm("There is no term " + termName + " in dictionary.");
-            return mapper.writeValueAsString(term);
+            CustomInfo infoMsg = new CustomInfo('i',"There is no term " + termName + " in dictionary.");
+            return mapper.writeValueAsString(infoMsg);
        }
```